src/vs/workbench/contrib/format/browser/formatActionsMultiple.ts

```diff
@@ -209,11 +209,14 @@ async function showFormatterPick(accessor: ServicesAccessor, model: ITextModel,
 		};
 	});
 
+	// auto focus the default formatter
+	let autoFocusItem = picks.filter((pick) => pick.description)[0];
```

Comment:
this is not nice/proper as it builds on a side-effect of a default formatter having a description. it would be better to use the loop above and the if check on line 208

Reply:
That does make sense, thanks for the feedback. I added a new commit.